Drivers/Src/Contactors.c

```diff
@@ -0,0 +1,66 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+
+/**
+ * Source file to mimic GPIO port communication
+ */ 
+
+#include "Contactors.h"
+
+/**
+ * @brief   Initializes contactors to be used
+ *          in connection with the Motor and Array
+ * @param   contactor the contactor to initialize
+ * @return  None
+ */ 
+void Contactors_Init(contactor_t contactor) {
```

Comment:
Is there any reason for this function to take an argument?

Reply:
In the event that we would want to initialize one contactor but not the other, I left this here. It likely won't be used in that way, but it keeps it consistent with the other drivers as well.

Reply:
^^
Both contactors are in the same port, so is it because once we actually interface with the actual board we'll also set up each pin in the port individually?

Reply:
Yeah we'll end up initializing each pin as an output pin, so we could end up initializing them separately.